docs/evaluation/eval-repo.md

```diff
@@ -0,0 +1,49 @@
+# New Repository Proposal: eval
+
+## Summary
+
+This document proposes a new repository under the `instructlab` GitHub organization:
+
+- `instructlab/eval`
+
+## Background
+
+The `instructlab/instructlab` repository currently includes no real implementation
+of Evaluation as described by the [LAB paper](https://arxiv.org/abs/2403.01081). The
+closest implementation currently in `instructlab/instructlab` via the `ilab test` command.
+
+`ilab test` as of this writing is only implemented for macOS with M-series chips. It uses
```

Comment:
I think gaps in `ilab` aren't worth mentioning because the library in the evaluation repo won't just be used by the CLI. This paragraph and the mention of ilab test in the previous paragraph and the next can be cut.

Reply:
I think it's relevant because at this time, it's the only related functionality visible anywhere and it will need to (re)moved as part of this work.

Reply:
Agreed with Russell - it's good for context and will be here if we ever need to go back and justify decisions around removing things